lib/assets/javascripts/twine.js.coffee

```diff
@@ -258,47 +267,47 @@ Twine.bindingTypes =
     {refresh, teardown}
 
   'bind-show': (node, context, definition) ->
-    fn = wrapFunctionString(definition, '$context,$root', node)
+    fn = wrapFunctionString(definition, '$context,$root,$modules', node)
     lastValue = undefined
     return refresh: ->
-      newValue = !fn.call(node, context, rootContext)
+      newValue = !fn.call(node, context, rootContext, modules)
       return if newValue == lastValue
       $(node).toggleClass('hide', lastValue = newValue)
 
   'bind-class': (node, context, definition) ->
-    fn = wrapFunctionString(definition, '$context,$root', node)
+    fn = wrapFunctionString(definition, '$context,$root,$modules', node)
     lastValue = {}
     return refresh: ->
-      newValue = fn.call(node, context, rootContext)
+      newValue = fn.call(node, context, rootContext, modules)
       for key, value of newValue when !lastValue[key] != !value
         $(node).toggleClass(key, !!value)
       lastValue = newValue
 
   'bind-attribute': (node, context, definition) ->
-    fn = wrapFunctionString(definition, '$context,$root', node)
+    fn = wrapFunctionString(definition, '$context,$root,$modules', node)
```

Comment:
Or eval. I can't think of a situation off the top of my head where we'd need it on other things but it might not hurt to also include it there. Hmm hmm hmm